server/main.lua

```diff
@@ -1755,7 +1755,8 @@ local function giveCitationItem(src, citizenId, fine, incidentId)
 		officer = OfficerFullName,
 	}
 	Player.Functions.AddItem('mdtcitation', 1, false, info)
-	TriggerClientEvent('QBCore:Notify', src, PlayerName.." received a citation!")
+	TriggerClientEvent('QBCore:Notify', src, PlayerName.." (" ..citizenId.. ") received a citation!")
+	if Config.QBManagementUse then exports['qb-management']:AddMoney(Officer.PlayerData.job.name, fine) end

```

Comment:
For pure readability, the if statement and its content should be split into three lines.

Reply:
oh i wanted to make it less messier

Reply:
done tho <3